templates.json

```diff
@@ -0,0 +1,149 @@
+[
+    {
+        "name": "csharp",
+        "source": "openfaas",
+        "description": "Official C# template",
+        "repo": "https://github.com/openfaas/templates",
+        "official": "true"
+    },
+    {
+        "name": "dockerfile",
+        "source": "openfaas",
+        "description": "Official Dockerfile template",
+        "repo": "https://github.com/openfaas/templates",
+        "official": "true"
+    },
+    {
+        "name": "go-armhf",
+        "source": "openfaas",
+        "description": "Official Golang on armhf template",
+        "repo": "https://github.com/openfaas/templates",
+        "official": "true"
+    },
+    {
+        "name": "go",
+        "source": "openfaas",
+        "description": "Official Golang template",
+        "repo": "https://github.com/openfaas/templates",
+        "official": "true"
+    },
+    {
+        "name": "java8",
+        "source": "openfaas",
+        "description": "Official Java 8 template",
+        "repo": "https://github.com/openfaas/templates",
+        "official": "true"
+    },
+    {
+        "name": "node-arm64",
+        "source": "openfaas",
+        "description": "Official NodeJS on arm64 template",
+        "repo": "https://github.com/openfaas/templates",
+        "official": "true"
+    },
+    {
+        "name": "node",
+        "source": "openfaas",
+        "description": "Official NodeJS 8 template",
+        "repo": "https://github.com/openfaas/templates",
+        "official": "true"
+    },
+    {
+        "name": "php7",
+        "source": "openfaas",
+        "description": "Official PHP 7 template",
+        "repo": "https://github.com/openfaas/templates",
+        "official": "true"
+    },
+    {
+        "name": "python-armhf",
+        "source": "openfaas",
+        "description": "Official Python 2 armhf template",
+        "repo": "https://github.com/openfaas/templates",
+        "official": "true"
+    },
+    {
+        "name": "python",
+        "source": "openfaas",
+        "description": "Official Python 2 template",
+        "repo": "https://github.com/openfaas/templates",
+        "official": "true"
+    },
+    {
+        "name": "python3-armhf",
+        "source": "openfaas",
+        "description": "Official Python 3 armhf template",
+        "repo": "https://github.com/openfaas/templates",
+        "official": "true"
+    },
+    {
+        "name": "python3",
+        "source": "openfaas",
+        "description": "Official Python 3 x86-64 template",
+        "repo": "https://github.com/openfaas/templates",
+        "official": "true"
+    },
+    {
+        "name": "ruby",
+        "source": "openfaas",
+        "description": "Official Ruby x86-64 template",
+        "repo": "https://github.com/openfaas/templates",
+        "official": "true"
+    },
+    {
+        "name": "node10-express-template",
```

Comment:
This is the repo name. What is the name in the `template.yml` file? - I think it's `node10-express`

So I should be able to do `faas template store pull node10-express` or fully qualified: `faas template store pull openfaas-incubator/node10-express-template`

Reply:
Yeah it is `node10-express` but not sure what you mean here, can we pull single folder from repository? Or you mean to read the template.yml and pull the whole repository if the single template that we need exists?

Reply:
This is now referenced ^